[
  {
    "signature": "rejectedExecution(java.lang.Runnable r, java.util.concurrent.ThreadPoolExecutor executor)",
    "name": "rejectedExecution",
    "containingClass": {
      "qualifiedName": "java.util.concurrent.RejectedExecutionHandler",
      "name": "RejectedExecutionHandler",
      "isArray": false
    },
    "targetClass": "java.util.concurrent.RejectedExecutionHandler",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "void",
      "name": "void",
      "isArray": false
    },
    "parameters": [
      {
        "type": {
          "qualifiedName": "java.lang.Runnable",
          "name": "Runnable",
          "isArray": false
        },
        "name": "r"
      },
      {
        "type": {
          "qualifiedName": "java.util.concurrent.ThreadPoolExecutor",
          "name": "ThreadPoolExecutor",
          "isArray": false
        },
        "name": "executor"
      }
    ],
    "protocol": {
      "member": "rejectedExecution(java.lang.Runnable r, java.util.concurrent.ThreadPoolExecutor executor)",
      "comment": "Method that may be invoked by a ThreadPoolExecutor when method_0 execute cannot accept a task. This may occur when no more threads or queue slots are available because their bounds would be exceeded, or upon shutdown of the Executor. In the absence of other alternatives, the method may throw an unchecked RejectedExecutionException, which will be propagated to the caller of execute.",
      "kind": "FreeText",
      "condition": ""
    }
  }
]